Editor/Credentials/Strings.cs

```diff
@@ -136,5 +136,21 @@ public static class Strings
         public const string TabAnywhere = "TabAnywhere";
         public const string TabEC2 = "TabEC2";
         public const string TabHelp = "TabHelp";
+        
+        public const string LabelHelpTitle = "LabelHelpTitle";
+        public const string LabelHelpTitleDescription = "LabelHelpTitleDescription";
+        public const string LabelHelpReportIssues = "LabelHelpReportIssues";
+        public const string LabelHelpDocumentation = "LabelHelpDocumentation";
+        public const string LabelHelpVideoTutorials = "LabelHelpVideoTutorials";
+        public const string LabelHelpAwsForum = "LabelHelpAwsForum";
+        public const string LabelHelpEstimatingTitle = "LabelHelpEstimatingTitle";
+        public const string LabelHelpEstimatingDescription = "LabelHelpEstimatingDescription";
+        public const string LabelHelpFleetIqTitle = "LabelHelpFleetIqTitle";
+        public const string LabelHelpFleetIqDescription = "LabelHelpFleetIqDescription";
+        public const string LabelHelpFlexMatchTitle = "LabelHelpFlexMatchTitle";
+        public const string LabelHelpFlexMatchDescription = "LabelHelpFlexMatchDescription";
+        public const string LabelEstimatingHelpLearnMore = "LabelEstimatingHelpLearnMore";
+        public const string LabelFleetIqHelpLearnMore = "LabelFleetIqHelpLearnMore";
+        public const string LabelFlexMatchHelpLearnMore = "LabelFlexMatchHelpLearnMore";
```

Comment:
Should this instead be `LabelHelpEstimatingLearnMore` / `LableHelpFleetIqLearnMore` / `LabelHelpFlexMatchLearnMore` to be consistent with the variables on lines 146, 147